tasks/create_github_deployment.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: "Create Github deployment"
+  uri:
+    url: "{{ wimpy_github_deployments_api }}"
+    method: POST
+    headers:
+      Authorization: "Token {{ wimpy_github_token }}"
+      Content-Type: "application/json"
+    body:
+      ref: "{{ wimpy_release_version }}"
+      environment: "{{ wimpy_deployment_environment }}"
+      description: "Deployment by Wimpy"
+    status_code: 201
+    body_format: json
+  register: wimpy_github_deployment_out
+  ignore_errors: yes
+
+- name: "Pending Deployment status"
+  uri:
+    url: "{{ wimpy_github_deployments_api }}/{{ wimpy_github_deployment_out.json.id }}/statuses"
+    method: POST
+    headers:
+      Authorization: "Token {{ wimpy_github_token }}"
+      Content-Type: "application/json"
+    body:
+      state: "pending"
+      target_url: ""
+      description: "Deployment started"
+    body_format: json
+    status_code: 201
+    ignore_errors: yes
+    when: wimpy_github_deployment_out|succeeded
```

Comment:
Ignoring errors and executing only if the task creating the deployment succeeded